### ISSUE TITLE
Express Checkout - Review page buttons inactive..??

### ISSUE THREAD

**Issue by angelleye** [angelleye/paypal-woocommerce#280]
I had this happen and showed you a video where the Place Order button was simply inactive on the review page if I did not have the Terms and Conditions set.  

You made some adjustments and that seems to have resolved the issue for me, however, I am getting multiple reports where people are saying this is happening to them on our 1.1.6.3.6 release.  

We need to look into this more and figure out what's causing it.

**Comment by angelleye** [comment]
This support ticket has login credentials for a site where this is happening:  https://www.angelleye.com/support/scp/tickets.php?id=195

They have WC guest checkout enabled and no T&C page set, so those are the same variables that was causing it for me before we made an adjustment.  It's not happening to me anymore, but I've had 2 or 3 people tell me this is happening to them.

**Comment by angelleye** [comment]
Here's another one where this is happening and the user provided more details.  Scroll to the end of the ticket...

https://www.angelleye.com/support/scp/tickets.php?id=181

**Comment by angelleye** [comment]
Here is yet another report of this same issue:  https://wordpress.org/support/topic/review-order-page-doesnt-work-sometimes?replies=2#post-7342594

He says his customers are complaining that it happens but he can't reproduce it either.  Something is definitely going on here, though.  It's the same that we were able to reproduce in #275 and we seemingly resolved, but there's obviously still a problem with what we did.

We need to test all the different configurations with T&C and Guest Checkout in WooCommerce, and we also need to test in all browsers.

**Comment by angelleye** [comment]
I have now been able to reproduce this.  The following setting causes the issue.

WooCommerce -> Settings -> General Tab
    
![image](https://cloud.githubusercontent.com/assets/629241/9485658/6f890fc0-4b82-11e5-822a-058bba9c92b6.png)

If I set Geolocate without page caching it doesn't happen.  Same with the other options.  Only the Geolocate with caching seems to cause the Place Order button on our review page to become disabled.
